src/library/librarycontrol.cpp

```diff
@@ -793,6 +816,31 @@ void LibraryControl::slotMoveFocus(double v) {
             QEvent::KeyPress, key, Qt::NoModifier, QString(), false, times});
 }
 
+void LibraryControl::slotMoveTrackForward(double v) {
+    if (v > 0) {
+        slotMoveTrack(1);
+    }
+}
+
+void LibraryControl::slotMoveTrackBackward(double v) {
+    if (v > 0) {
+        slotMoveTrack(-1);
+    }
+}
+
+void LibraryControl::slotMoveTrack(double v) {
```

Comment:
Please add a  /// comment here explaining that these moves equal Alt + Up/Down.

Reply:
done